isaac2/Assets/Scenes/SampleScene.unity

```diff
@@ -370,11 +453,11 @@ Rigidbody2D:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 893021813}
-  m_BodyType: 1
+  m_BodyType: 0
```

Comment:
I had body type set to kinematic so that Player and Enemy wouldn't push each other around

Reply:
oops. I changed it back.